### ISSUE TITLE
Terrain Generation Bugs

### ISSUE THREAD

**Issue by Mad-Gnome** [zmatez/wildnature#59]
Minecraft 1.14.4
Wild Nature: 2.1.6.1
Forge: 28.2.3

Not sure if this is intended or not, but getting some odd 'second layer' jungles.
![WildNatureJungle](https://user-images.githubusercontent.com/23144159/77365895-7fde2c00-6d1c-11ea-9671-93cc87b1ff75.png)

Polders biome has some interesting terrain generation still.
![WildNaturePolders](https://user-images.githubusercontent.com/23144159/77365823-5fae6d00-6d1c-11ea-9d5e-b7010429aa63.png)

**Comment by zmatez** [comment]
Yeah, I will have to edit current terrain generator, some bugs it has. For that jungle you could only create new world, its rare issue, depends on seed. And polders need fix of course

**Comment by Dakedres** [comment]
Also getting these weird stripes across the world in some "biomes", although the stripes themselves are different biomes.
![image](https://media.discordapp.net/attachments/614152515323297803/696149402187858026/unknown.png?width=645&height=363)

**Comment by zmatez** [comment]
Any other mods you use? These strips aren't generating in plain wildnature..

**Comment by Username2-stack** [comment]
I get the issue with the strips as well, though I do have a lot of other mods.
